### PR TITLE
Added _mm_unpackhi_ps function

### PR DESCRIPTION
**Pull request by mohanrajendran**
I have added the `_mm_unpackhi_ps` function to `sse.rs`. This is my first pull request. Appreciate your feedback.

### CODE REVIEW

File: src/x86/sse.rs [hunk 1]
```diff
@@ -51,7 +61,7 @@ pub fn _mm_movemask_ps(a: f32x4) -> i32 {
 }
 
 #[allow(improper_ctypes)]
-extern {
+extern "C" {
```
Comment (BurntSushi) [inline src/x86/sse.rs]: What motivated this change?

File: src/x86/sse.rs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 use v128::*;
 
+use simd_llvm::simd_shuffle4;
```
Comment (BurntSushi) [inline src/x86/sse.rs]: Please group this with the other crate imports in alphabetical order. :-) So it should be:

```rust
use simd_llvm::simd_shuffle4;
use v128::*;
```

Thanks!

Comment (mohanrajendran) [reply src/x86/sse.rs]: Will do!

File: src/x86/sse.rs (hunk 1 above)
Comment (mohanrajendran) [reply src/x86/sse.rs]: Looks like my rustfmt pass made this change. There is an option to turn this off. I'll do that.

**Comment by BurntSushi** [comment]
Fantastic, thank you!

**Pull request merged**